src/main/java/gov/nasa/cumulus/metadata/aggregator/UMMGranuleFile.java

```diff
@@ -423,10 +423,13 @@ private JSONObject exportSpatial() throws ParseException{
         spatialExtent.put("HorizontalSpatialDomain", horizontalSpatialDomain);
 
         if (granule instanceof IsoGranule) {
-            String polygon = ((IsoGranule) granule).getPolygon();
-            if (polygon != "" && polygon != null) {
+            String polygonString = ((IsoGranule) granule).getPolygon();
+            if (polygonString != "" && polygonString != null) {
```

Comment:
Please use apache lang3 StringUtils.isEmpty to accomplish this very same logic.  It is null proof. 
Hopefully not string with only space but it could be "    "

hence.  StringUtils.isEmpty(StringUtils.trim(polygonString)) will do

I like the way the variable is named.  Thumb up on that

Reply:
Sounds good - updated in the next commit.